packages/shared/lib/core/nfts/actions/updateNftInAllAccountNfts.ts

```diff
@@ -10,12 +10,14 @@ export function updateNftInAllAccountNfts(accountIndex: number, nftId: string, p
         const nft = state[accountIndex].find((_nft) => _nft.id === nftId)
         if (nft) {
             const downloadUrl = nft.downloadUrl
-            void getIpfsUri({ hash: downloadUrl }).then((ipfsUri) => {
-                if (ipfsUri) {
-                    nft.downloadUrl = ipfsUri
-                    nft.composedUrl = ipfsUri
-                }
-            })
+            if (downloadUrl) {
```

Comment:
I was thinking something along the lines like this check we have in explorer: https://github.com/iotaledger/explorer/blob/dev/client/src/helpers/hooks/useNftMetadataUri.ts#L22
Looks to me that we now try to `getIpfsUri` even if `downloadUrl` is not ipfs link.